src/sentry/static/sentry/app/components/button.tsx

```diff
@@ -24,7 +24,7 @@ type Props = {
   busy?: boolean;
   to?: string | object;
   href?: string;
-  icon?: string;
+  icon?: string | object;
```

Comment:
Is the object form a react node? If so you should use React.ReactNode.

Reply:
This works